dashboard/app/views/api/teacher_announcement.html.haml

```diff
@@ -1,27 +1,28 @@
 - teacher_blog_url = 'http://teacherblog.code.org/'
 - scaling_url = 'http://teacherblog.code.org/post/152968866984/an-important-note-about-computer-science'
 - mc_url = 'https://code.org/minecraft'
+- code_studio_video_url = 'https://www.youtube.com/watch?v=ZRNRVu4DFcs'
```

Comment:
Our video lightbox with non-youtube fallback acts funky on the teacher dashboard. I'm not sure why, but I believe because of angular. It was getting complicated to make it work on this page so in the interest of keeping the change simple this at least makes the video available in some form.